distributor-api-v1/operations.md

```diff
@@ -208,15 +208,15 @@ Preferred initial call used to obtain all static data about distributor configur
 | `DisplaySpecialRequests` | boolean | optional | Determines whether to display special requests field during checkout. |
 | `Enterprise` | [Enterprise](operations.md#enterprise) | required | Enterprise to which the `Configuration` belongs. |
 | `OnlineTravelAgencies` | array of string | required | Array of travel agencies to include in comparison banner. |
-| `PaymentCardInput` | string [PaymentCardInput](operations.md#paymentcardinput) | required | Determines how to handle payment cards. |
+| `PaymentCardRequirement` | string [PaymentCardRequirement](operations.md#paymentcardrequirement) | required |  Determines how to handle payment cards. |
 | `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Form fields which are required and need to be filled in. |
 | `ServiceId` | string | required | Unique identifier of the service to which the configuration is bound to. |
 
-#### PaymentCardInput
-
-* `NotRequested` - Payment card info is not requested.
-* `Requested` - Payment card info is requested, but not validated.
-* `Required` - Payment card info is requested and validated. 
+#### PaymentCardRequirement
+* `NotRequired` - Payment card info is never required. 
+* `AlwaysRequired` - Payment card info is always requested and validated.
```

Comment:
why are we mixing together the required and requested word? :)
we should use only one to make it clear.